src/__tests__/attributes.js

```diff
@@ -116,9 +116,9 @@ test('attribute selector with hex escape', '[ng\\00003acloak]', (t, tree) => {
 test('assign attribute name requiring escape', '[ng\\:cloak]', (t, tree) => {
     let attr = tree.nodes[0].nodes[0];
     attr.attribute = "ng:foo";
-    t.deepEqual(attr.raws.attribute, 'ng\\3A foo');
+    t.deepEqual(attr.raws.attribute, 'ng\\:foo');
     t.deepEqual(attr.attribute, 'ng:foo');
-    t.deepEqual(tree.toString(), '[ng\\3A foo]');
+    t.deepEqual(tree.toString(), '[ng\\:foo]');
```

Comment:
hm, looks something invalid (title `assign attribute name requiring escape`), but now no escaping

Reply:
It's still escaped 👍 it's taking `ng:foo` and converting it to `ng\:foo`. Previously it was converting that to `ng\3A foo` but the `:` character only needs to be escaped that way in IE < 8. In IE9+ (and every other browser), it's safe to escape `:` as `\:`.

The main motivation for this change is to workaround the bug in css-loader (https://github.com/webpack-contrib/css-loader/issues/578) where the leading slash is removed when it is followed by a hex character. This change is an attempt to make it possible to use as many non-standard characters as possible by not unnecessarily escaping `:` as `\3A ` (which gets converted to `3A ` by css-loader) and instead escaping it as `\:` which css-loader leaves intact 👍